src/cbc_sdk/platform/legacy_alerts.py

```diff
@@ -342,8 +366,10 @@ def set_time_range(self, key, **kwargs):
         Returns:
             AlertSearchQuery: This instance.
         """
-        if key not in ["create_time", "first_event_time", "last_event_time", "last_update_time", "backend_timestamp", "backend_update_timestamp"]:
-            raise ApiError("key must be one of create_time, first_event_time, last_event_time, backend_timestamp, backend_update_timestamp, or last_update_time")
+        if key not in ["create_time", "first_event_time", "last_event_time", "last_update_time", "backend_timestamp",
```

Comment:
Should we be mentioning the new/v7 fields names (e.g. `last_update_timestamp`) or because these are the legacy alerts we should not?

Reply:
This will be addressed with cbapi-4979 - support v7 time range structure.